aws/app/lambda/reliability/reliability.js

```diff
@@ -41,12 +43,12 @@ exports.handler = async function (event) {
      Form submission object contains:
        formID - ID of form,
        language - form submission language "fr" or "en",
-       submission - submission type: email, vault
        responses - form responses: {formID, securityAttribute, questionID: answer}
        form - Complete Form Template
+       deliveryOption - user choice on whether the submission should be sent by email or saved to the vault (`{ emailAddress: string; emailSubjectEn: string; emailSubjectFr: string }`)
     */
-
-    if (formSubmission.submission.vault) {
+   
+    if (!formSubmission.deliveryOption?.emailAddress || formSubmission.deliveryOption.emailAddress === "") {
```

Comment:
Could we not just test to see if the `deliveryOption` exists?  I would assume that to switch between the Vault and the Notify processing we would delete the `DeliveryOption` object associtated with the Template.